app/controllers/kalibro_configurations_controller.rb

```diff
@@ -74,10 +75,14 @@ def kalibro_configuration_params
     params[:kalibro_configuration]
   end
 
+  def attributes_params
+    params[:attributes][:public] == "1"
+  end
```

Comment:
Why are we always setting the public field as 1? Can't we have private configurations?